docs/docs/tutorial/index.md

```diff
@@ -335,54 +315,44 @@ hydra-node gen-hydra-key --output-file credentials/bob-hydra
 ```
 
 </TabItem>
-</Tabs>
+If you are collaborating with another individual, exchange the verification (public) keys: `{alice,bob}-node.vk` and `{alice,bob}-hydra.vk` to ensure secure communication.
 
-If you are doing this tutorial with a friend, now is the time to exchange the
-verification (public) keys: `{alice,bob}-node.vk` and `{alice,bob}-hydra.vk`.
-You can use any authenticated communication channel for this where you can be
-sure your peer cannot be easily impersonated.
+If you are collaborating with a partner, exchange the verification (public) keys: `{alice,bob}-node.vk` and `{alice,bob}-hydra.vk` to ensure secure communication
+Before launching the `hydra-node`, it's crucial to establish and communicate each participant's network connectivity details. This includes the IP addresses and ports where `Alice` and `Bob's` nodes will be reachable for layer 2 network interactions. For this tutorial, we're using placeholder IP addresses and ports which should be replaced with your actual network details:
 
-Besides keys, we also want to communicate each other's connectivity information.
-That is, an IP address / hostname + port where we will be reachable for our
-layer two network using `hydra-node`. For the purpose of this tutorial we are
-assuming an IP address and port for `alice` and `bob` which works on a single
-machine, but please replace usages below with your respective addresses:
+Before launching the `hydra-node`, it's crucial to establish and communicate each participant's network connectivity details. This includes the IP addresses and ports where `Alice` and `Bob's` nodes will be reachable for the layer two network interactions. For this tutorial, we're using placeholder IP addresses and ports which should be replaced with your actual network details:
 
 <!-- TODO: can we make peers configurable via some text input? -->
 
 Alice: <code>127.0.0.1:5001</code>
 
-Bob: <code>127.0.0.1:5001</code>
```

Comment:
Should this line be deleted, @oduameh?

Reply:
Thanks for flagging this @olgahryniuk I accidentally deleted this line